navigator/backend/src/main/scala/com/digitalasset/navigator/UIBackend.scala

```diff
@@ -56,6 +56,7 @@ abstract class UIBackend extends LazyLogging with ApplicationInfoJsonSupport {
     * (supply a default)
     */
   def defaultConfigFile: Path = Paths.get("ui-backend.conf")
+  def defaultAccessTokenFile: Path = Paths.get(sys.props("user.home"), ".daml", "secret")
```

Comment:
The location of this file could be read using the `ProjectConfig` library.

See https://github.com/digital-asset/daml/blob/master/daml-assistant/scala-daml-project-config/src/main/scala/com/digitalasset/assistant/config/ProjectConfig.scala for the scala library (which currently only supports the project config and not the global daml config).

See https://github.com/digital-asset/daml/blob/master/daml-assistant/daml-project-config/DA/Daml/Project/Consts.hs#L43 and https://github.com/digital-asset/daml/blob/master/daml-assistant/daml-project-config/DA/Daml/Project/Config.hs#L38 for the original Haskell library.

Reply:
`ProjectConfig` is about the `daml.yaml` file, where we should not store the token (since that is supposed to be checked in with the rest of your code).

After a short conversation with @associahedron and @hurryabit we agreed on using `$DAML_HOME/secret` as a well-known location (`$DAML_HOME` is `~/.daml` on Mac/Linux), while leaving the possibility to pass a different location on the command line.

On top of that, I'll probably make an effort to try to read from the expected well-known location if `$DAML_HOME` is not found, unless you have something against it.

So basically something along these lines:
```scala
// Uses $DAML_HOME from the SDK Assistant or falls back to its default location:
// - on Windows: %APPDATA%/daml
// - on Mac and Linux: ~/.daml
// The file called `secret` will be looked in the resolved location
private[this] val defaultAccessTokenFile: Path =
  sys.env
    .get("DAML_HOME")
    .fold(if (sys.props("os.name").toLowerCase.startsWith("win")) {
      Paths.get(sys.env("APPDATA"), "daml")
    } else {
      Paths.get(sys.props("user.home"), ".daml")
    })(Paths.get(_))
    .resolve("secret")
```